internal/handlers/msg_service_router.go

```diff
@@ -139,6 +144,20 @@ func (msr *PioMsgServiceRouter) RegisterService(sd *grpc.ServiceDesc, handler in
 				return nil, err
 			}
 
+			if msr.circuitBreaker != nil {
+				msgURL := sdk.MsgTypeURL(req)
+
+				var isAllowed bool
```

Comment:
Noticed that this was defined separately. Can probably get rid of this unless this is just a copy/paste of their code.

Reply:
We use different linter settings than the SDK, and their version of this code doesn't pass our linter. So I'd either need to make this change or add a `//nolint` comment. I couldn't think of a good reason to ignore the linter though, so I split it out like this.

What I have here:
```golang
var isAllowed bool
isAllowed, err = msr.circuitBreaker.IsAllowed(ctx, msgURL)
```

What they have:
```golang
isAllowed, err := msr.circuitBreaker.IsAllowed(ctx, msgURL)
```

What our linter says with what they have:
```
internal/handlers/msg_service_router.go:150:16: shadow: declaration of "err" shadows declaration at line 131 (govet)
				isAllowed, err := msr.circuitBreaker.IsAllowed(ctx, msgURL)
				           ^
```